codecov.yml

```diff
@@ -2,6 +2,8 @@ codecov:
   require_ci_to_pass: yes
   notify:
     after_n_builds: 3
+  ignore:
+    - "**/*__init__.py"
```

Comment:
Should this be `**/__init__.py`? Do we intend to ignore files named like `mcbaz__init__.py`?

Reply:
I don't think this even works as it stands. I also tried `**/__init__.py` and it still incorrectly flagged up the `templates/__init__.py`. I'll see if I can fix it.

Reply:
I see you've removed this for now.